_test/test_experiment/test_experiment_methods.m

```diff
@@ -35,7 +35,8 @@
             exp(3).psi = 10;            
             exp(3).angular_units = 'rad';                        
 
-            obj.sample_exper= Experiment(IX_detector_array,instruments,samples,exp);
+            %obj.sample_exper= Experiment(IX_detector_array,instruments,samples,exp);
+            obj.sample_exper= Experiment([],instruments,samples,exp);
```

Comment:
Is an empty array equivalent to an empty detector array?

Reply:
no, this is a placeholder for existing functionality. It will be replaced in PR 999